### DISCUSSION TITLE
Adding and updating more data

### DISCUSSION THREAD

**Discussion by nickmalbsn** [kyleskom/NBA-Machine-Learning-Sports-Betting#91] [Ideas]
Is there a way to add rosters, injuries, trades, current stats, etc. to the project in order to get a more accurate prediction of which team is going to win? I am brand new to coding so idrk much. Is this something that is possible to add?

**Comment by kyleskom** [comment]
Yes there is a way and it is possible. I don't think this task is suitable to a new programmer.
